index.php

```diff
@@ -1,9 +0,0 @@
-<?php
```

Comment:
@adamf321 @Fcog  why this file was deleted? When I go to Appearance -> Themes I get this error:
`Template is missing. Standalone themes need to have a index.php template file.`

Can I bring it back?

Reply:
yes please @nelsonamaya82 !